packages/blade/src/components/ActionList/ActionListFooter.tsx

```diff
@@ -8,18 +8,15 @@ import { getActionListFooterRole } from './getA11yRoles';
 import BaseBox from '~components/Box/BaseBox';
 import type { IconComponent } from '~components/Icons';
 import { useDropdown } from '~components/Dropdown/useDropdown';
-import {
-  isReactNative,
-  makeSize,
-  makeAccessible,
-  MetaConstants,
-  metaAttribute,
-  isValidAllowedChildren,
-} from '~utils';
+import { isReactNative } from '~utils';
 import { Text } from '~components/Typography';
 import { useBottomSheetAndDropdownGlue } from '~components/BottomSheet/BottomSheetContext';
-import type { TestID } from '~src/_helpers/types';
-import { assignWithoutSideEffects } from '~src/utils/assignWithoutSideEffects';
+import type { TestID } from '~utils/types';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+import { makeSize } from '~utils/makeSize';
+import { makeAccessible } from '~utils/makeAccessible';
+import { isValidAllowedChildren } from '~utils/isValidAllowedChildren';
+import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
```

Comment:
This seems a bit verbose to me now: 

This was much cleaner
```
import {
  isReactNative,
  makeSize,
  makeAccessible,
  MetaConstants,
  metaAttribute,
  isValidAllowedChildren,
} from '~utils';
```

Why was the imports changed like this?

@saurabhdaware thoughts?

Reply:
so there are internal utilities which are only used inside our repository and there are external utilities which we are getting ready to be used by everyone.

external utilities are the ones that are re-exported from `utils/index.ts`. So far we've been re-exporting a lot of utilities to consumers (even the ones which are supposed to be internal only)

Agree about the imports not looking good at some places. Was kind of in favour of having `utils/internal/` directory but shouldn't be required because of auto-imports (we discussed it here https://docs.google.com/document/d/1P0_8lmqLsk--sbroJ-4y5ZwKTs0VmODsSg2u001mqLM/edit?disco=AAAAv_DS3xc)

Can move them to internal directory in future if required